### PR TITLE
websites: az webapp up supports Python

### LINKED ISSUE #259: [az webapp up] Add support for Python in az webapp up
### Extension name

az webapp

### Description of issue (in as much detail as possible)

The issue pertains to adding `az webapp up` support for `Python|3.7` on App Service on Linux. 

### Workflow [P0]

When a user calls the `az webapp up -n <unique_app_name>` command, 

1. Determine if the current directory contains a `requirements.txt` or `.py` file.
    _If yes_ proceed to step 2
    _else_ do not identify as python and throw an error

2. Create a default resource group. 

3. Create a default App Service Plan on Linux (`--is-linux`). 

4. Create a Python app (`--runtime "PYTHON|3.7"`) with the specified name.

5. Zip deploy files from the current working directory to the web app.

### Note [P1]

During step 5, determine if the project contains a directory with the following structure: 

```
(dir_name) (e.g. venv)
| bin/
| include/
| lib/
```

If so, please don't include this directory in the deployment.

### PR DESCRIPTION
**Pull request by panchagnula**
Fixes #259 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/webapp/azext_webapp/_constants.py [hunk 1]
```diff
@@ -6,12 +6,14 @@
 NODE_VERSION_DEFAULT = "8.9"
 NETCORE_VERSION_DEFAULT = "2.0"
 DOTNET_VERSION_DEFAULT = "4.7"
+PYTHON_VETSION_DEFAULT = "3.7"
```
Comment (panchagnula) [inline src/webapp/azext_webapp/_constants.py]: Need to get confirmation from PM on this,

File: src/webapp/azext_webapp/create_util.py [hunk 2]
```diff
@@ -71,6 +73,9 @@ def get_runtime_version_details(file_path, lang_name):
     elif lang_name.lower() == STATIC_RUNTIME_NAME:
         version_detected = "-"
         version_to_create = "-"
+    elif lang_name.lower() == PYTHON_RUNTIME_NAME:
+        version_detected = "-"
```
Comment (panchagnula) [inline src/webapp/azext_webapp/create_util.py]: Need to follow up with PM on available version for Linux & detecting this reliably

Comment (panchagnula) [reply src/webapp/azext_webapp/create_util.py]: Follow-up with PM: Only a single version is supported right now so this is good

File: src/webapp/azext_webapp/create_util.py [hunk 3]
```diff
@@ -17,7 +17,9 @@
     DOTNET_RUNTIME_NAME,
     DOTNET_VERSION_DEFAULT,
     DOTNET_VERSIONS,
-    STATIC_RUNTIME_NAME)
+    STATIC_RUNTIME_NAME,
+    PYTHON_RUNTIME_NAME,
+    PYTHON_VETSION_DEFAULT)
```
Comment (williexu) [inline src/webapp/azext_webapp/create_util.py]: typo, i think you mean VERSION?

Comment (panchagnula) [reply src/webapp/azext_webapp/create_util.py]: Thanks for catching this. Will update this.

**Pull request merged**